.github/fabricbot.json

```diff
@@ -3687,7 +3687,7 @@
             {
               "name": "bodyContains",
               "parameters": {
-                "bodyPattern": "(aot)/i",
+                "bodyPattern": "([Aa][Oo][Tt])+",
```

Comment:
What kind of regex engine is being used? .NET? If it is, could this be written:

```suggestion
                "bodyPattern": "(?i)(?:aot)",
```

Reply:
We clarified offline that they don't support options. 😢

Reply:
How else do you do case insensitive? is there a parameter for it? Seems like a major limitation.

Reply:
It is a major limitation indeed. Which they now know about !